Wizardry/Core/BattleSys/Source/PreBattleCalc.c

```diff
@@ -616,6 +616,45 @@ void PreBattleCalcAuraEffect(struct BattleUnit * attacker, struct BattleUnit * d
                     attacker->battleAttack  += 3;
 #endif
 
+#if (defined(SID_DriveStr) && (SID_DriveStr < MAX_SKILL_NUM))
+                if (SkillTester(unit, SID_DriveStr) && range2[i] == 1)
+                {
+                    if (!IsMagicAttack(attacker))
+                    attacker->battleAttack  += 4;
```

Comment:
Indentation issue

Reply:
Should be good now. Sorry again for the disagrement, I'm gonna spent some time with my IDE style review so that it doesn't prop up again